openapi/SwarmCommon.yaml

```diff
@@ -933,6 +933,23 @@ components:
           items:
             $ref: "#/components/schemas/StatusSnapshotResponse"
 
+    StatusNeighborhoodResponse:
+      type: object
+      properties:
+        address:
+          type: string
+        chunkCount:
```

Comment:
Is this `reserveSize` or `reserveSizeWithinRadius` or something completely different?

Reply:
This field should be called `reserveSizeWithinRadius`. Good call.

Reply:
renamed